pytensor/bin/pytensor_cache.py

```diff
@@ -24,7 +24,7 @@
 
 def print_help(exit_status):
     if exit_status:
-        print(f"command \"{' '.join(sys.argv)}\" not recognized")
+        print(f'command "{" ".join(sys.argv)}" not recognized')
```

Comment:
Don't you still need different quotes inside?
```suggestion
        print(f'command "{' '.join(sys.argv)}" not recognized')
```

Reply:
yeah, these are automatic changes, I need to investigate